master/internal/agent/proto.go

```diff
@@ -0,0 +1,73 @@
+package agent
+
+import (
+	"github.com/golang/protobuf/ptypes/timestamp"
+
+	"github.com/determined-ai/determined/master/pkg/container"
+	"github.com/determined-ai/determined/master/pkg/device"
+	proto "github.com/determined-ai/determined/master/pkg/proto/agentv1"
+)
+
+func toProtoAgent(a agentSummary) *proto.Agent {
```

Comment:
question: this seems a bit unfortunate, this is for backwards compatibility right?

Reply:
Ya, this is for backwards compatibility. Once the project wraps up, we will deprecate the old API and we can remove these old types.